cdk/README.md

```diff
@@ -0,0 +1,46 @@
+# Kafka Connector for Amazon EventBridge Deployment on AWS
+
+This CDK stack is sample deployment to for the Kafka Connector for Amazon EventBridge. 
+
+> :warning: **This sample is not meant to be deployed to production environments!**
+
+## Overview
+
+This sample deploys several components in your AWS account:
+
+- Amazon MSK Serverless Cluster
+- Amazon S3 Bucket
+- IAM Roles & Security Groups
+- VPC & related components
+- Custom Amazon EventBridge Bus
+- Amazon ECS cluster with 2 Services (Data Producer & Transaction Analyzer)
+- (Optional) Amazon MSK Connector
+
+To learn more about those components please read the blog on the [AWS Big Data Blog](https://aws.amazon.com).
+
+![Architecture Diagram](./images/architecture.png)
+
+## Deployment
+
+There are 2 deployment options for the CDK stack:
+
+To deploy everything mentioned above, inlcuding the Connector itself please run the following command in this directory:
+```shell
+cdk deploy --context deploymentMode=FULL
+```
+
+To deploy only the Prerequisites and later deploy the connector manually. Please run the following command in this directory:
```

Comment:
It would be helpful to add a line on when PREREQ will be preferred option.

Reply:
Done